bootstraps/stage1/spack.in.yaml

```diff
@@ -0,0 +1,28 @@
+# This is a Spack Environment file.
+#
+# It describes a set of packages to be installed, along with
+# configuration settings.
+spack:
+  specs:
+  - binutils +stage0
+  - gcc +stage0 +binutils ~bootstrap languages=c,c++
```

Comment:
Should `gcc` be pinned here?

Reply:
Because of the other envs yes it should, good point.